frontend/src/app/components/ExperimentsListPage.js

```diff
@@ -13,7 +13,8 @@ export default class ExperimentsListPage extends React.Component {
     super(props);
 
     let showEmailDialog = false;
-    if (cookies.get('first-run')) {
+    if (cookies.get('first-run') ||
+        window.location.search.indexOf('utm_campaign=restart-required') >= -1) {
```

Comment:
This is stupidly hacky, but our cookies are all session cookies.

CC @ckprice because it's in his realm of things.